examples/virtual-dom-wip/platform/Html/Internal/Server.roc

```diff
@@ -74,6 +79,7 @@ insertRocScript = \document, initData, wasmUrl, hostJavaScript ->
         initData
         |> Encode.toBytes Json.toUtf8
         |> Str.fromUtf8
+        |> Result.mapErr toU64BadUtf8Err
```

Comment:
Oh wow this is hilarious! When you said I was encoding a Nat to JSON I didn't think of here!
This is a bug. I don't want a Result here at all. This line should really be `|> Result.withDefault ""`. The whole thing is probably full of stupid bugs from not having been able to run it.

Reply:
😆 Yeah I figured maybe error isn't exactly what you want. Feel free to patch on this branch if you want to change the semantics.

Reply:
done